src/__experimental__/components/select/select.component.js

```diff
@@ -355,9 +355,6 @@ class Select extends React.Component {
   }
 
   inputIcon(typeAhead) {
-    if (this.props.inputIcon) {
```

Comment:
@harpalsingh can you check this behaviour, it's not defined in the DLS.

![image](https://user-images.githubusercontent.com/2328042/80134788-2f662280-8597-11ea-99c8-c2546a67355c.png)

Reply:
Yes, as you mentioned, this is not defined in the DLS site, so need to understand why it was being done.